telethon/utils.py

```diff
@@ -1123,3 +1123,17 @@ def __getattr__(self, item):
             return w.fget(self.wrapped)
         else:
             return w
+
+
+def stripped_to_photo(stripped):
+    """
+    adds the jpg header and footer to a stripped image.
+    transcoded from https://github.com/telegramdesktop/tdesktop/blob/bec39d89e19670eb436dc794a8f20b657cb87c71/Telegram/SourceFiles/ui/image/image.cpp#L225
+    """
+    if len(stripped) < 3 or stripped[0] != '\x01':
+        return stripped
```

Comment:
Should we rather `raise` an error indicating that it wasn't valid? Or rather `return b''`, because writing this will fail.

Reply:
I think we should just write the bytes as is..